examples/opf/tools/sp_plotter.py

```diff
@@ -260,10 +259,11 @@ def testSP():
       for i in xrange(inputSize):
         time.sleep(0.001)
         if iter == numIter - 1:
-          outputs[i] = sp.compute(inputs[i], learn=doLearn, infer=False)
+          sp.compute(inputs[i], learn=doLearn, activeArray=outputs[i])
```

Comment:
btw, OT for this PR, but different implementations (cpp/py of SP/TP) behave inconsistently, 
sometimes it's:
* `output = xx.compute(input, params)`
* `xx.compute(input, params, output)`

For interoperability of the implementations, we should unify that. (as I come from java, i'd prefer the first approach, no biggie though)